<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>.,?!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>abc</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>def</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>⇐</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>jkl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>210</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>mno</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>ghi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>tuv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>wxyz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>pqrs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>⬆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_13">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>space</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>410</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>701</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>39</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
